src/persistence/net.jl

```diff
@@ -0,0 +1,66 @@
+
+"""
+Writes a graph `g` to a file `f` in the [Pajek
+NET](http://gephi.github.io/users/supported-graph-formats/pajek-net-format/) format.
+Returns 1 (number of graphs written).
+"""
+function savenet(f::IO, g::SimpleGraph, gname::AbstractString = "g")
+    println(f, "*Vertices $(nv(g))")
+    # write edges
+    if is_directed(g)
+        println(f, "*Arcs")
+    else
+        println(f, "*Edges")
+    end
+    for e in edges(g)
+        println(f, "$(src(e)) $(dst(e))")
+    end
+    return 1
+end
+
+
+"""Reads a graph from file `fname` in the [Pajek
+ NET](http://gephi.github.io/users/supported-graph-formats/pajek-net-format/) format.
+ Returns 1 (number of graphs written).
+"""
+function loadnet(f::IO, gname::AbstractString = "g")
+    line =readline(f)
+    # skip comments
+    while startswith(line, "%")
+        line =readline(f)
+    end
+    n = parse(Int, matchall(r"\d+",line)[1])
+    for fline in eachline(f)
+        line = fline
+        (ismatch(r"^\*Arcs",line) || ismatch(r"^\*Edges",line)) && break
+    end
+    if ismatch(r"^\*Arcs",line)
+        g = DiGraph(n)
+    else
+        g = Graph(n)
+    end
+    while ismatch(r"^\*Arcs",line)
+        for fline in eachline(f)
+            line = fline
+            orin
```

Comment:
@CarloLucibello what is `orin`? It's causing things to fail on master for some reason.